FabiWare/fabi.h

```diff
@@ -26,7 +26,7 @@
 
 #define VERSION_STRING "FABI v2.5"
 
-//#define DEBUG_OUTPUT      //  if debug output is desired
+#define DEBUG_OUTPUT      //  if debug output is desired
```

Comment:
Debug should only be turned on for debugging purpose - it can cause misbehaviour in the GUI because of the debug messages